app/domain/models/adopter.py

```diff
@@ -50,7 +74,8 @@ def adopter_helper(adopter) -> dict:
             "birthdate": adopter["birthdate"],
             "gender": adopter["gender"],
             "email": adopter["email"],
-            "animal": adopter["animal"],
+            "photo": adopter["photo"],
+            "animals": adopter["animals"],
```

Comment:
Acredito que retornar todos animais q o usuario tem sempre q buscar por ele não seja a melhor prática, ainda mais porque esse vetor será um vetor de IDS e não os proprios animais, tendo entao q sempre tratar os IDS. em um contexto real essa request ficaria pesada e muito requisitada, não apenas para obter o usuario (adotante) mas para obter os animais do mesmo que faz parte de outra parte do sistema.

Recomendo remover e criar uma rota a parte, mas Ok por enquanto devido ao tempo de desenvolvimentos q temos.

fica a sugestão do TODO:
```suggestion
            # TODO: Remover isso e criar uma rota para obter animais do adotante
            "animals": adopter["animals"],
```